src/blockchain/ain/jellyfish/utxo-provider.service.ts

```diff
@@ -31,11 +36,16 @@ export class UtxoProviderService {
     whaleService.getClient().subscribe((client) => (this.whaleClient = client));
   }
 
-  async getCurrentNumberOfUnspent(address: string): Promise<number> {
+  async getStatistics(address: string): Promise<UtxoStatistics> {
     if (!this.unspent.has(address)) {
       await this.retrieveUnspent(address);
     }
-    return this.unspent.get(address)?.length ?? 0;
+    const unspent = this.unspent.get(address);
+    const quantity = unspent?.length ?? 0;
+    const sortedUnspent = unspent?.sort((a, b) => UtxoProviderService.orderDescending(a, b));
+    const biggest = sortedUnspent?.length ?? 0 > 0 ? new BigNumber(sortedUnspent[0].vout.value) : undefined;
```

Comment:
Could use undefined here:
```ts
const biggest = new BigNumber(sortedUnspent?.[0]?.vout.value);
```
This will return `NaN` (can remove `| undefined` in interface). And `NaN.gte(1000)` is still false.

Example:
![image](https://user-images.githubusercontent.com/85513542/198338610-16a9bc21-6c09-45e2-8c9b-67276c4731c3.png)